lib/screens/event_detail_screen/event_details.dart

```diff
@@ -0,0 +1,174 @@
+import 'package:cached_network_image/cached_network_image.dart';
+import 'package:flutter/cupertino.dart';
+import 'package:flutter/material.dart';
+
+class details_page extends StatefulWidget {
+  const details_page({Key? key}) : super(key: key);
+
+  @override
+  State<details_page> createState() => _details_pageState();
+}
+
+class _details_pageState extends State<details_page> {
+  @override
+  Widget build(BuildContext context) {
+    return Scaffold(
+      backgroundColor: Colors.white,
+      body: NestedScrollView(
+        headerSliverBuilder: (BuildContext context, bool innerBoxIsScrolled){
+          return<Widget>[
+            SliverAppBar(
+              shape: const RoundedRectangleBorder(
+                borderRadius: BorderRadius.only(
+                  bottomLeft: Radius.circular(20.0),
+                  bottomRight: Radius.circular(20.0), ), ),
+
+              expandedHeight: 300,
+              floating: true,
+              pinned: true,
+              flexibleSpace:
+
+              FlexibleSpaceBar(
+
+                centerTitle: true,
+
```

Comment:
Run `flutter format .` to format the project properly and use auto formatter in the IDE